tests/ttnn/unit_tests/operations/test_moreh_dot_backward.py

```diff
@@ -135,11 +128,69 @@ def test_moreh_matmul_1d_backward(input_shape, requires_grad, device):
         assert passing
 
     if require_other_grad:
-        ttcpu_other_grad = tt_other_grad.cpu().to(cpu_layout).unpad_from_tile(input_shape).to_torch()
+        ttcpu_other_grad = ttnn.to_torch(tt_other_grad)
 
         passing, output_pcc = comp_allclose_and_pcc(
             torch_other.grad, ttcpu_other_grad.reshape(-1), pcc=0.999, rtol=rtol, atol=atol
         )
         logger.debug(f"other_grad passing={passing}")
         logger.debug(f"other_grad pcc={output_pcc}")
         assert passing
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    (
+        [1, 1, 1, 10],  # test not mutiple of 32 case
+        [1, 1, 1, 32],  # test single tile
+        [1, 1, 1, 352],  # test multiple tiles
+        [1, 1, 1, 323],  # test multiple tiles, not a multiple of 32
+    ),
+)
+@pytest.mark.parametrize(
+    "requires_grad",
+    (
+        (True, False),
+        (False, True),
+        (True, True),
```

Comment:
```suggestion
        [True, False],
        [False, True],
        [True, True],
```
Should use list instead of tuple as HyungSuk reminded before.

Reply:
done